lib/modules/manager/gomod/artifacts.ts

```diff
@@ -113,14 +116,23 @@ function addAuthFromHostRule(
 
 function getUpdateImportPathCmds(
   updatedDeps: PackageDependency[],
-  { constraints, newMajor }: UpdateArtifactsConfig
+  { constraints }: UpdateArtifactsConfig
 ): string[] {
   const updateImportCommands = updatedDeps
-    .map((dep) => dep.depName!)
-    .filter((x) => !x.startsWith('gopkg.in'))
+    .map(({ depName, newVersion }) => ({
+      depName: depName!,
+      newMajor: major(newVersion!),
+    }))
+    // Skip path upates going from v0 to v1
+    .filter(
+      ({ depName, newMajor }) => !depName.startsWith('gopkg.in') && newMajor > 1
```

Comment:
```suggestion
      ({ depName, newMajor }) =>
        !depName.startsWith('gopkg.in') && newMajor && newMajor > 1
```

Handle the case of `newMajor` being `null` due to `semver.valid()` returning `null`.

This might need prettier formatting after this change, I'm suggesting this by hand